packages/components/src/DateTimePickers/shared/components/IconButton/IconButton.component.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import Icon from '../../../../Icon';
+import theme from './IconButton.scss';
+
+function IconButton(props) {
+	const {
+		icon: iconProps,
+		className: buttonClassNameProp,
+		...buttonPropsRest
+	} = props;
+
+	const {
+		className: iconClassNameProp,
+		...iconPropsRest
+	} = iconProps;
+
+	const buttonClassName = classNames(buttonClassNameProp, theme.btn);
+	const iconClassName = classNames(iconClassNameProp, theme.icon);
```

Comment:
Useless constants. classNames can be used directly in the return.

Reply:
More or less like for the others variables remarks.
Here extracting those informations from the JSX, give the possibility to think of the main structure and props without needing to dive directly in the exact data.